percy/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Percy
-  VERSION = '0.0.8'.freeze
+  VERSION = '0.0.9.beta.0'.freeze
```

Comment:
```suggestion
  VERSION = '0.0.9.beta0'.freeze
```
thats correct format as per the link pradum shared